proposals/0003_documentation-strategy.md

```diff
@@ -0,0 +1,130 @@
+
+| website |
+| ------- | 
+| https://2.docs.brightway.dev/ |
+| https://brightway.dev/ (redirects to the above) |
+| https://brightway-docs.readthedocs.io/ |
+
+The primary documentation previously at https://2.docs.brightway.dev/ was difficult to navigate for first-time users. It does not differentiate clearly between different user needs and offers no interactivity. In addition, various notebooks written as part of training events and seminars are collected separately at [brightway-lca/brightway2](https://github.com/brightway-lca/brightway2/tree/master/notebooks) and at [brightway-lca/from-the-ground-up](https://github.com/brightway-lca/from-the-ground-up). Some are outdated (Jupyter Notebook version) and can no longer be viewed in GitHub. An alltogether confusing state of affairs!
+
+A solution has been proposed by @tngTUDOR in the form of the Diataxis Framework. This framework was created and developed by Daniele Procida during his tenure at Divio. It formalizes different types of technical documentation according to the different needs of users. It is widely adopted in software development (e.g. at [Canonical, Cloudflare, Google, etc.](https://diataxis.fr/adoption/)).
+
+|    | serve our study | serve our work |
+| -: | :-------------: | :------------: |
+practical steps | __Tutorials__ <br/> _learning-oriented_ | __How-To Guides__ <br/> _task-oriented_ |
+theoretical knowledge | __Explanation__ <br/> _understanding-oriented_ | __Reference__ <br/> _information-oriented_ |
+
+Table 1: The [Diataxis framework](https://diataxis.fr/) 
+
+The framework makes an [important distinction based on the needs of the user](https://diataxis.fr/tutorials-how-to/): The purpose of a _tutorial_ is to provide a successful _learning experience_. The purpose of a _how-to guide_ is to to help the user _accomplish a task_.
+
+## Proposal
+
+All code added to the Brightway code base shall include docstrings conforming to the [Numpy Style Guide](https://numpydoc.readthedocs.io/en/latest/format.html). Existing code that does not conform to this standard shall be considered legacy code and shall be refactored. New code that does not conform to this standard shall not be merged into the Brightway code base.
+
+Based on the Diataxis framework, all Brightway documentation shall be structured as follows:
+
+|    | serve our study | serve our work |
+| -: | :-------------: | :------------: |
+practical steps | __Tutorials (=E-Book)__ <br/> Pages at <br/>  [`learn.brightway.dev`](https://github.com/brightway-lca/brightway-training) | __How-To Guides__ <br/> "Gallery" at <br/> [`docs.brightway.dev`](https://github.com/brightway-lca/brightway-documentation) |
+theoretical knowledge | __Explanation__ <br/> "LCA" at <br/> [`docs.brightway.dev`](https://github.com/brightway-lca/brightway-documentation)| __Reference__ <br/> "API Reference" at <br/>  [`docs.brightway.dev`](https://github.com/brightway-lca/brightway-documentation) |
```

Comment:
blocking: the BEP suggest in table 2 that we use "LCA" for the theory. The current main branch of the documentation does not have this anymore. I believe it's "Overview" that has replaced this. @michaelweinold  I suggest we update the BEP to say "Overview" instead of LCA.